public/app/features/canvas/elements/icon.tsx

```diff
@@ -7,7 +7,8 @@ import { SanitizedSVG } from 'app/core/components/SVG/SanitizedSVG';
 import { getPublicOrAbsoluteUrl } from 'app/features/dimensions';
 import { DimensionContext } from 'app/features/dimensions/context';
 import { ColorDimensionEditor, ResourceDimensionEditor } from 'app/features/dimensions/editors';
-import { APIEditorConfig, callApi } from 'app/plugins/panel/canvas/editor/element/APIEditor';
+import { APIEditorConfig } from 'app/plugins/panel/canvas/editor/element/APIEditor';
+import { callApi } from 'app/plugins/panel/canvas/editor/element/utils';
```

Comment:
Is this really needed/supported for icon?

Reply:
looks like it is for the onClick handler 🤷🏻

Reply:
at this stage, i would keep it out of icon -- i *think* it existed as a cheap button.

So i vote we remove it for now

Reply:
_cheap button_ :sweat_smile: I'll remove it for now, we'll bring it back if needed.